nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/test/java/org/apache/nifi/csv/TestJacksonCSVRecordReader.java

```diff
@@ -69,7 +70,7 @@ public void testUTF8() throws IOException, MalformedRecordException {
         fields.add(new RecordField("name", RecordFieldType.STRING.getDataType()));
         final RecordSchema schema = new SimpleRecordSchema(fields);
 
-        try (final InputStream bais = new ByteArrayInputStream(text.getBytes());
+        try (final InputStream bais = new ByteArrayInputStream(text.getBytes(StandardCharsets.UTF_8));
```

Comment:
Does this fail without the fix above? Wondering if we should have another unit test that specifies a different charset

Reply:
> Does this fail without the fix above? Wondering if we should have another unit test that specifies a different charset

It would fail if the test were run in a process with `-Dfile.encoding` other than UTF-8.  I noticed it on the Github CI (Windows node):
- https://github.com/greyp9/nifi/runs/5843639230?check_suite_focus=true

Good call; another test might be useful.  Of the choices, ISO_8859_1 would probably work best:
- https://docs.oracle.com/javase/7/docs/api/java/nio/charset/StandardCharsets.html

I'll have a look.